### PR TITLE
Delist coins with failing electrum

### PR DESCRIPTION
**Pull request by smk762**
ref: https://github.com/KomodoPlatform/coins/issues/498#issuecomment-1264241192

This has been announced in discord and those involved have been pinged. 
Will keep this as draft for the next week or two for them to respond, and merge just before next release.

### CODE REVIEW

**Comment by michelvankessel** [comment]
Please delist USDI. The project seems to be dead.

**Comment by WikiMin3R** [comment]
Artbyte electrumx servers (2 of 4) flagged should now be back online. Please do not remove. They were temporarily down.

**Comment by smk762** [comment]
> Artbyte electrumx servers (2 of 4) flagged should now be back online. Please do not remove. They were temporarily down.
> 
> ```
>   },
>   {
>     "url": "electrumx-three.artbyte.live:50012",
>     "protocol": "SSL",
>     "disable_cert_verification": false,
>     "ws_url": "electrumx-three.artbyte.live:50013"
>   },
>   {
>     "url": "electrumx-four.artbyte.live:50012",
>     "protocol": "SSL",
>     "disable_cert_verification": false,
>     "ws_url": "electrumx-four.artbyte.live:50013"
> ```

Thanks for info, I've added these two back in. If it helps, I have a script here which can be modified to use half a dozen hardcoded electrums for monitoring purposes. I'll add telegram alert integration later, and maybe we can setup a channel for electrum maintainers to subscribe to.

File: assets/config/0.5.6-coins.json [hunk 1]
```diff
@@ -8579,16 +8844,10 @@
...
     "nomics_id": "ZET",
-    "electrum": [
-      {
+    "electrum": [{
         "url": "207.180.252.200:50011",
         "protocol": "TCP",
         "ws_url": "207.180.252.200:50013"
-      },
-      {
-        "url": "zeta-seed-c.zetacoin.tech:50012",
```
Comment (cipig) [inline assets/config/0.5.6-coins.json:8589]: this one is back online

File: assets/config/0.5.6-coins.json [hunk 2]
```diff
@@ -3826,6 +3969,24 @@
...
+        "disable_cert_verification": false,
+        "protocol": "SSL"
+      },
+      {
+        "url": "electrum3.cipig.net:20067",
+        "disable_cert_verification": false,
+        "protocol": "SSL"
+      },
+      {
+        "url": "spv17.lbry.com:50001"
```
Comment (cipig) [inline assets/config/0.5.6-coins.json:3989]: electrums from lbry.com don't work properly... please remove

**Comment by smk762** [comment]
Closing - once https://github.com/KomodoPlatform/atomicDEX-Desktop/pull/2000 is merged and we source from directly the coins repo, this PR will be obsolete.

**Pull request closed without merging**